src/routes/+layout.js

```diff
@@ -0,0 +1,24 @@
+import Feature from "../components/Feature.svelte";
+import Grid from "../components/Grid.svelte";
+import Page from "../components/Page.svelte";
+import Teaser from "../components/Teaser.svelte";
+import { apiPlugin, storyblokInit, useStoryblokApi } from "@storyblok/svelte";
+
+/** @type {import('./$types').LayoutLoad} */
+export async function load() {
+  storyblokInit({
+    accessToken: "TxjwDdrurDCNvQwkIyREVwtt",
+    use: [apiPlugin],
+    components: {
+      feature: Feature,
+      grid: Grid,
+      page: Page,
+      teaser: Teaser,
+    },
```

Comment:
Do we have to include the option for the US region here as well? (commented out, but to make people aware)

Reply:
added! thank you for highlight this.